sdk/java-sdk/src/main/java/kalix/javasdk/workflowentity/WorkflowEntity.java

```diff
@@ -334,13 +399,16 @@ public AsyncCallBuilder(String name,  Function<I, CompletionStage<O>> callFunc)
         this.callFunc = callFunc;
       }
 
+      /**
+       * Transition to a next step based on step action result.
+       *
+       * @param transitionFunc Function that transform the action result to a {@link Effect.TransitionalEffect}
+       * @return ???
```

Comment:
`???` is missing?

Reply:
yeah, sorry, a work in progress, I should put this PR as a draft, but it is fixed now